cg/meta/workflow/nf_analysis.py

```diff
@@ -252,6 +253,15 @@ def get_paired_read_paths(self, sample=Sample) -> tuple[list[str], list[str]]:
         )
         return fastq_forward_read_paths, fastq_reverse_read_paths
 
+    def get_unmapped_bam_read_paths(self, sample=Sample) -> list[str]:
+        """Gather BAM file path for a sample based on the BAM tag."""
+        return [
+            hk_file.full_path
+            for hk_file in self.housekeeper_api.files(
+                bundle=sample.internal_id, tags={AlignmentFileTag.BAM}
+            )
+        ]
```

Comment:
In general i like working with `Path` objects instead of strings in these cases. Is that an easy fix or does it require further changes?

```suggestion
        return [
            Path(hk_file.full_path)
            for hk_file in self.housekeeper_api.files(
                bundle=sample.internal_id, tags={AlignmentFileTag.BAM}
            )
        ]
```

Reply:
will test now 👍